src/plugins/azgenerator/azgenerator.ts

```diff
@@ -12,6 +12,16 @@ export async function processRequest(host: Host) {
     const debug = await host.GetValue('debug') || false;
     try {
         const session = await startSession<CodeModel>(host, {}, codeModelSchema);
+        const azureCliFolder = await host.GetValue("azure-cli-folder");
+        if (!isNullOrUndefined(azureCliFolder)) {
+            if (isNullOrUndefined(session.model.language['az'])) {
+                session.model.language['az'] = {}
+            }
+            session.model.language['az']['azureCliFolder'] = azureCliFolder;
+        }
+
+        const azOutputFolder = await host.GetValue('az-output-folder');
```

Comment:
>az-output-folder [](start = 52, length = 16)

what's the difference with az-output-folder and output-folder? can you reuse "output-folder" in the below?

Reply:
az-output-folder is from readme.az.md, it' sonly for az. we no longer support output-folder because autorest.python's output-folder will impact ours.

Reply:
won't fix